### PR TITLE
Properly quote the changelog entry in /vc command

### PR DESCRIPTION
**Pull request by dcermak**

### CODE REVIEW

File: .github/workflows/vc-command.yml [hunk 1]
```diff
@@ -53,7 +53,7 @@ jobs:
               -vvvv --os-version $OS_VERSION \
```
Comment (dirkmueller) [inline .github/workflows/vc-command.yml:53]: while being at it, how about quoting this as well?

File: .github/workflows/vc-command.yml [hunk 2]
```diff
@@ -53,7 +53,7 @@ jobs:
...
                 add_changelog_entry \
                 --user $user \
```
Comment (dirkmueller) [inline .github/workflows/vc-command.yml:55]: and here?

File: .github/workflows/vc-command.yml (hunk 1 above)
Comment (dcermak) [reply .github/workflows/vc-command.yml:53]: This one will never contain any spaces

File: .github/workflows/vc-command.yml (hunk 2 above)
Comment (dcermak) [reply .github/workflows/vc-command.yml:55]: Usernames should not contain spaces so it shouldn't be necessary :crossed_fingers:

Comment (dirkmueller) [reply .github/workflows/vc-command.yml:55]: quoting handles more than just spaces (for example usernames like '-miska-' are valid, but need to be quoted in shell)

**Pull request merged**